doc/licenses/apache-commons-codec-1.15/README

```diff
@@ -0,0 +1,8 @@
+Apache Commons Codec (http://commons.apache.org/proper/commons-codec/)
+-----------------------------------------------------------------------
+
+    Version: 1.15
+    From: 'Apache Software Foundation' (http://www.apache.org)
```

Comment:
Sure - I can update this where it occurs real quick.

The URL within `NOTICE` was updated only because it's a verbatim copy of the `NOTICE` provided by the library, but `http://www.apache.org/` redirects to `https://www.apache.org/`. Makes sense to follow that intent.